### PR TITLE
[AV-1837] Added export command

### PR DESCRIPTION
**Pull request by dasconnor**
Added the `key export` command. Also renamed the `DeleteKeys` function to `DeleteKey` since it only deletes one key.

### CODE REVIEW

File: cmd/keycmd/export.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	keyName := args[0]
+
+	keyPath := app.GetKeyPath(keyName)
+	keyBytes, err := os.ReadFile(keyPath)
+	if err != nil {
+		return err
+	}
+
+	if filename == "" {
+		fmt.Println(string(keyBytes))
```
Comment (holisticode) [inline cmd/keycmd/export.go:44]: Note:This is not necessarily printing the key in hex encoded format, it just prints the content of the file. As we are writing the hex-encoded version of the key on `Save`, this implicitly holds. Nevertheless, wanted to have mentioned it.

File: tests/e2e/key/suite.go [hunk 2]
```diff
@@ -117,4 +121,61 @@ var _ = ginkgo.Describe("[Key]", func() {
...
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Check two keys are equal
+		equal, err := utils.CheckKeyEquality(testKey, outputKey)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(equal).Should(gomega.BeTrue())
```
Comment (holisticode) [inline tests/e2e/key/suite.go:179]: Shouldn't we remove the test output file afterwards?

**Review by holisticode — COMMENTED** [review]
Basically LGTM, a couple of nits I guess

**Review by felipemadero — APPROVED** [review]
LGTM

File: tests/e2e/key/suite.go (hunk 2 above)
Comment (dasconnor) [reply tests/e2e/key/suite.go:179]: it gets cleaned up by the `AfterEach`

File: cmd/keycmd/export.go (hunk 1 above)
Comment (dasconnor) [reply cmd/keycmd/export.go:44]: I think this is fine for now. If we want to be a little more flexible, we can add more support in the future.

**Pull request merged**